TCAT/Utilities/Extensions.swift

```diff
@@ -372,28 +372,24 @@ func presentShareSheet(from view: UIView, for route: Route, with image: UIImage?
 /** Bold a phrase that appears in a string, and return the attributed string.
     Only shows the last bolded phrase.
  */
-func bold(pattern: String, in string: String) -> NSMutableAttributedString {
-    let fontSize = UIFont.systemFontSize
-    let fontAttribute = [NSAttributedString.Key.font : UIFont.systemFont(ofSize: fontSize)]
+func bold(pattern: String, in string: String, from font: UIFont, to boldFont: UIFont) -> NSMutableAttributedString {
```

Comment:
it'd be nice if this could be an extension in one of the string classes. that way its not a global func

Reply:
also for args i think having longer args names helps explain what the args are for. 

So:
- `pattern` -> `textToBeBold`
- `in` -> `containerText`
- `from` -> `originalFont`
-  `boldFont` think is good